src/main/java/uk/gov/hmcts/divorce/document/content/GenerateJudicialSeparationCORefusedForAmendmentCoverLetter.java

```diff
@@ -75,22 +108,33 @@ private Map<String, Object> templateContent(final CaseData caseData, final Long
 
         templateContent.put(IS_JOINT, !caseData.getApplicationType().isSole());
 
-        templateContent.put(FIRST_NAME, applicant.getFirstName());
-        templateContent.put(LAST_NAME, applicant.getLastName());
-        templateContent.put(ADDRESS, applicant.getAddress());
+        templateContent.put(FEEDBACK, conditionalOrderCommonContent.generateLegalAdvisorComments(caseData.getConditionalOrder()));
 
-        if (caseData.getDivorceOrDissolution().isDivorce()) {
-            templateContent.put(MARRIAGE_OR_CIVIL_PARTNERSHIP, MARRIAGE);
-            templateContent.put(DIVORCE_OR_CIVIL_PARTNERSHIP, DIVORCE);
+        if (caseData.getApplication().isPaperCase() && applicant.isRepresented()) {
```

Comment:
Same again, isPaperCase check not required?

Reply:
Resolved